lib/lte_link_control/lte_lc.c

```diff
@@ -506,6 +507,26 @@ static void at_handler_mdmev(const char *response)
 	event_handler_list_dispatch(&evt);
 }
 
+static void at_handler_rai(const char *response)
+{
+	int err;
+	struct lte_lc_evt evt = {0};
+
+	__ASSERT_NO_MSG(response != NULL);
+
+	LOG_DBG("%%RAI notification");
+
+	err = parse_rai(response, &evt.rai);
+	if (err) {
+		LOG_ERR("Can't parse RAI notification, error: %d", err);
+		return;
+	}
+
+	evt.type = LTE_LC_EVT_RAI_UPDATE;
```

Comment:
I assume that modem only sends the `%RAI` notification when something has changed? So, no need to have a check here?

Reply:
This notification is sent when RRC connection is established.

Reply:
Ok, so should the event be sent only when something has changed? LTE LC has this kind of checks for example for `LTE_LC_EVT_NW_REG_STATUS` and `LTE_LC_EVT_CELL_UPDATE`.

Reply:
And doesn't have those checks for `LTE_LC_EVT_EDRX_UPDATE` and `LTE_LC_EVT_RRC_UPDATE`

Reply:
At least `+CSCON` is never sent by the modem unless the value has changed. Most likely it's the same for `+CEDRXP` as well. Let's discuss this offline.

Reply:
It seems that modem already does some filtering, so no filtering is needed in LTE LC.